src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/util/BlackDuckResponseCache.java

```diff
@@ -121,7 +124,7 @@ public String getSeverity(String vulnerabilityUrl) {
             logger.debug("Error fetching vulnerability view", e);
         }
 
-        return severity;
+        return severity.name();
```

Comment:
Could this throw an NPE?

Reply:
good call, if vulnerabilityView is an empty optional then it would throw an NPE.  how do you guys feel about returning inline conditionals?  or would you rather getSeverity return an Optional?